src/index.js

```diff
@@ -361,4 +361,13 @@ class Dayjs {
   }
 }
 
-export default config => (new Dayjs(config))
+function DayjsEntry(config) {
+  return new Dayjs(config)
+}
+
+DayjsEntry.unix = (unixTimestamp) => {
```

Comment:
I belive it's better to implement `unix` method like static function
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static
instead of writing this wrapper.

Reply:
I'm sorry. I think you are right! Apologize for this comment